.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+ci:
+  autofix_commit_msg: "[pre-commit.ci] auto fixes from pre-commit.com hooks"
+  autofix_prs: true
+
+repos:
+  - repo: local
+    hooks:
+      - id: isort
```

Comment:
Sollten wir nochmal checken, ob und wenn ja inwiefern sich die verschiedenen Tools gegenseitig beeinflussen? Hatte in Erinnerung, dass z.B. black manche Sachen so formatiert, dass flake meckert, oder dass black die Imports anders formatiert als es isort macht.

Reply:
https://sbarnea.com/lint/black/

Reply:
In theory, the `.flake8` file should take care of most of it. And in the `pyproject.toml` isort is set to be compatible with black. For now, we could try the settings and adjust them if needed.